Makefile

```diff
@@ -36,6 +36,15 @@ OUTPUT_DIR := $(abspath .output)
 # Self-contained GOPATH dir.
 GO_DIR := $(OUTPUT_DIR)/go
 
+# Base image used for all golang containers
+GOLANG_IMAGE := golang:1.20.8-bullseye
```

Comment:
Can we have docker build args for other images (e.g. gcr.io/gke-release/debian-base:bullseye-v1.4.3-gke.9)?
Then we just need to update this single file for image update.

Reply:
done